### PR TITLE
Fix kc db import on os

### PR DESCRIPTION
**Pull request by riuvshin**
added workarounds to adapt kc json templates to s2i build
fixes: https://github.com/eclipse/che/issues/7131

### CODE REVIEW

File: dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh [hunk 1]
```diff
@@ -46,8 +46,9 @@ if [ "${CHE_SERVER_URL}" == "" ]; then
     CHE_SERVER_URL="https://${CHE_SERVER_ROUTE_HOST}"
   fi
 fi
-
+DEFAULT_CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD=true
 oc set env buildconfig/keycloak-for-che CHE_SERVER_URL=${CHE_SERVER_URL}
+oc set env buildconfig/keycloak-for-che CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD=${CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD:-${DEFAULT_CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD}}
```
Comment (davidfestal) [inline dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh]: you should be able to set the 2 env variables on the same line.

```
Usage:
  oc set env RESOURCE/NAME KEY_1=VAL_1 ... KEY_N=VAL_N [options]
```

**Review by davidfestal — APPROVED** [review]

File: dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh (hunk 1 above)
Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh]: separator is just a space?

**Review by sleshchenko — APPROVED** [review]

File: dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh (hunk 1 above)
Comment (davidfestal) [reply dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh]: yes, it seems.

Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/multi-user/configure_and_start_keycloak.sh]: updated

**Review by skabashnyuk — APPROVED** [review]

**Review by garagatyi — APPROVED** [review]
Looks OK but not the best way of usage of puppet templates, though.

**Review by benoitf — APPROVED** [review]

**Review by l0rd — APPROVED** [review]

**Pull request merged**